node/packages/aws-lambda-otel-extension/test/benchmark/README.md

```diff
@@ -0,0 +1,199 @@
+# Benchmarks
+
+## Tested use cases
+
+### Node.js function ([`callback`](../../fixtures/lambdas/callback.js))
+
+No-op Node.js function. Handler takes callback and immediately invokes it with dummy payload
+
+### Node.js + Express function ([`express`](../../fixtures/lambdas/express.js))
+
+Simple express app with two endpoints which respond with dummy payload.
+Function is invoked with payload that reflecxts one that function will receive if it would be triggered through API Gateway integration
```

Comment:
```suggestion
Function is invoked with payload that reflects one that function will receive if it would be triggered through API Gateway integration
```